src/python/pants/help/help_json_schema.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from __future__ import annotations
+
+import re
+from typing import Any, Iterable
+
+from packaging.version import Version
+
+from pants.version import VERSION
+
+GENERATED_JSON_SCHEMA_FILENAME = f"pantsbuild-{VERSION}.json"
+DOCS_URL = "https://www.pantsbuild.org"
+VERSION_MAJOR_MINOR = f"{Version(VERSION).major}.{Version(VERSION).minor}"
```

Comment:
May as well import `MAJOR_MINOR`:
https://github.com/pantsbuild/pants/blob/ceaa203f94762477ac05b9a1fc747d60e5ddc29d/src/python/pants/version.py#L32-L35

Reply:
Oh of course, done!